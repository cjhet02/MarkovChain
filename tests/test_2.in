test2.txt
0
0
