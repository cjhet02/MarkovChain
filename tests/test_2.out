Enter cleaned data file name: how - 1: are - 100% 
are - 4: you - 50% the - 25% they - 25% 
you - 2: ? - 100% 
? - 3: where - 66.6667% what - 33.3333% 
where - 2: are - 100% 
the - 1: kids - 100% 
kids - 1: ? - 100% 
what - 1: are - 100% 
they - 1: doing - 100% 
doing - 1: ? - 100% 

Enter desired sentence length: 
Enter desired number of sentences: 